src/main/java/org/ambraproject/wombat/model/SearchFilter.java

```diff
@@ -23,23 +23,24 @@
 package org.ambraproject.wombat.model;
 
 import com.google.common.collect.ImmutableList;
-
-import java.util.LinkedHashSet;
+import com.google.common.collect.ImmutableSet;
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 public class SearchFilter {
 
-  private final ImmutableList<SearchFilterItem> searchFilterResult;
+  private final transient ImmutableList<SearchFilterItem> searchFilterResult;
```

Comment:
A late curiousity, but do you recall why you had to prevent serialization here?

Reply:
Just to save a little bandwidth. The list is partitioned into two lists with `activeFilterItems` and `inactiveFilterItems`. It probably wasn't necessary.